physics/module_sf_noahmp_glacier.f90

```diff
@@ -1464,6 +1464,11 @@ subroutine glacier_flux (nsoil     ,nsnow   ,emg     ,isnow   ,df      ,dzsnso
 ! 2m air temperature
      ehb2  = fv*vkc/(log((2.+z0h)/z0h)-fh2)
      cq2b  = ehb2
+! for opt_sfc 3
+     if (opt_sfc ==3) then
+       ehb2  = fv*vkc/fh2
```

Comment:
check for fh2~0?

Reply:
No, fh2 > 0.

Reply:
OK, I didn't see any limit checks in stability subroutine

Reply:
The stability becomes zero only under some conditions which we should avoid.